src/activities/drawletters/drawletters.svg

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
I have takes this image from :-
http://gcompris.net/incoming/lang/lang/words/draw.png